extensions/kubernetes/kind/deployment/src/main/java/io/quarkus/kind/deployment/KindProcessor.java

```diff
@@ -1,123 +1,37 @@
-        deploymentTargets.produce(
-                new KubernetesDeploymentTargetBuildItem(KIND, DEPLOYMENT, DEPLOYMENT_GROUP, DEPLOYMENT_VERSION,
-                        KIND_PRIORITY, true));
-
-        String name = ResourceNameUtil.getResourceName(config, applicationInfo);
-        resourceMeta.produce(
-                new KubernetesResourceMetadataBuildItem(KUBERNETES, DEPLOYMENT_GROUP, DEPLOYMENT_VERSION, DEPLOYMENT, name));
-    }
-
-    @BuildStep
-    public void createAnnotations(KubernetesConfig config, BuildProducer<KubernetesAnnotationBuildItem> annotations) {
-        config.getAnnotations().forEach((k, v) -> {
-            annotations.produce(new KubernetesAnnotationBuildItem(k, v, KIND));
-        });
-    }
-
-    @BuildStep
-    public void createLabels(KubernetesConfig config, BuildProducer<KubernetesLabelBuildItem> labels,
-            BuildProducer<ContainerImageLabelBuildItem> imageLabels) {
-        config.getLabels().forEach((k, v) -> {
-            labels.produce(new KubernetesLabelBuildItem(k, v, KIND));
-            imageLabels.produce(new ContainerImageLabelBuildItem(k, v));
-        });
-    }
-
-    @BuildStep
-    public List<ConfiguratorBuildItem> createConfigurators(KubernetesConfig config,
-            List<KubernetesPortBuildItem> ports) {
-        List<ConfiguratorBuildItem> result = new ArrayList<>();
-        KubernetesCommonHelper.combinePorts(ports, config).entrySet().forEach(e -> {
-            result.add(new ConfiguratorBuildItem(new AddPortToKubernetesConfig(e.getValue())));
-        });
-        return result;
-    }
-
-    @BuildStep
-    public List<DecoratorBuildItem> createDecorators(ApplicationInfoBuildItem applicationInfo,
-            OutputTargetBuildItem outputTarget,
-            KubernetesConfig config,
-            PackageConfig packageConfig,
-            Optional<MetricsCapabilityBuildItem> metricsConfiguration,
-            List<KubernetesAnnotationBuildItem> annotations,
-            List<KubernetesLabelBuildItem> labels,
-            List<KubernetesEnvBuildItem> envs,
-            Optional<BaseImageInfoBuildItem> baseImage,
-            Optional<ContainerImageInfoBuildItem> image,
-            Optional<KubernetesCommandBuildItem> command,
-            List<KubernetesPortBuildItem> ports,
-            Optional<KubernetesHealthLivenessPathBuildItem> livenessPath,
-            Optional<KubernetesHealthReadinessPathBuildItem> readinessPath,
-            List<KubernetesRoleBuildItem> roles,
-            List<KubernetesRoleBindingBuildItem> roleBindings,
-            Optional<CustomProjectRootBuildItem> customProjectRoot) {
-
-        return DevClusterHelper.createDecorators(KIND, applicationInfo, outputTarget, config, packageConfig,
-                metricsConfiguration, annotations, labels, envs, baseImage, image, command, ports, livenessPath, readinessPath,
-                roles, roleBindings, customProjectRoot);
-    }
```

Comment:
Is it expected that you remove all this code? Is the extension still doing something?

Reply:
Yes, this code was doing nothing anyway.